public/css/bootstrap-theme.css

```diff
@@ -0,0 +1,599 @@
+/*!
```

Comment:
You can use CDN script tags or a gem to include bootstrap... You don't have to copy it into your app...